platform/security/certificate/security-certificate-generator/src/main/java/org/codice/ddf/security/certificate/generator/CertificateCommand.java

```diff
@@ -82,12 +142,33 @@ public static String configureDemoCert(String commonName) {
      * private key is also stored in the entry.
      *
      * @param dn String params in the form {@code attrKey=attrVal} composing a distinguished name.
-     *           e.g. {@code configureDemoCertWithDN("cn=John Whorfin", "o=Yoyodyne", "l=San Narciso", "st=California", "c=US")}
+     *           e.g. {@code configureDemoCertWithDN(new String[] {"cn=John Whorfin", "o=Yoyodyne", "l=San Narciso", "st=California", "c=US"), null}
```

Comment:
Introduced because we had to overload the methods 😉 and I copied and pasted the javadoc.